source/cloud/nvidia/brev.md

```diff
@@ -4,41 +4,66 @@ review_priority: "p0"
 
 ![Screenshot of the "Create your first instance" UI](/_static/images/platforms/brev/brev1.png)
 
-Select "Container Mode".
+2. Select "Container Mode".
 
 ![Screenshot showing "Container Mode" highlighted](/_static/images/platforms/brev/brev2.png)
 
-Attach the "NVIDIA RAPIDS" Container.
+3. Attach the "NVIDIA RAPIDS" Container.
 
 ![Screenshot showing the "NVIDIA RAPIDS" container highlighted](/_static/images/platforms/brev/brev3.png)
 
-Configure your own instance.
+4. Configure your own instance.
 
 ![Screenshot of the "Create your Instance" UI with an NVIDIA T4 GPU highlighted](/_static/images/platforms/brev/brev4.png)
 
-And hit "Deploy".
+5. And hit "Deploy".
 
 ![Screenshot of the instance creation summary screen with the deploy button highlighted](/_static/images/platforms/brev/brev5.png)
 
+### 2. Setting up your Brev Launchable
+1. Click Create Launchable
+1. Click **Compute** and select your GPU based on GPU type, number of GPUs, the cloud provider, and/or budget you have.  It is good to understand your GPU requirements before picking an instance, or use it to figure out which instance.  Once selected, click **Save** 
+1. Click ** Container**.  When adding the **Container**, you can use the NVIDIA RAPIDS Container, use Docker Compose and edit our example yaml so that it preinstalls any additional conda or pip packages into your container before entry.  
+  1. If you just need standard NVIDIA RAPIDS install, Select **Container Mode > NVIDIA RAPIDS Container**. If using the Base Container, you may need to preinstall Jupyter.  If using the Notebooks Container, **Do not** Preinstall Jupyter, as that will break your instance.
+  1. If you need a customized environment, with additional packages on top of NVIDIA RAPIDSm use **Docker Compose** When using Docker Compose, you can upload a docker-compse yaml file.  Here is an example docker-compose file, **[docker/brev/docker-compose-nb-2412.yaml](https://github.com/clara-parabricks-workflows/single-cell-analysis-blueprint/raw/main/docker/brev/docker-compose-nb-2412.yaml)** that you can use as your base.  **Do not** Preinstall Jupyter when using that file, as it already will be installed.
+  1. Click **Save**
+1. Click **Files** and add in any publicly avaialble single file or  github repository.  Click **Save**
+1. In **Ports**, please open ports **8888, 8787, and 8786**.  Name port 8888 `jupyter` so Brev can treat it as a Jupyter-Lab based instance and provide an **Open Notebook** button.  Click **Save**
+1. Name your Launchable, then Save your Launchable!
+1. Whenever you're ready to use your Launchable, Select your Launchable and hit **Deploy Launchable** 
+
 ## Accessing your instance
+There are a few ways to access your instance:
+1. Directly access Jupyter Lab from the Brev GUI
+1. Using the Brev CLI to connect to your instance....
+  1. Using Visual Studio Code
+  1. Using SSH via your terminal
+1. Access using the Brev tunnel
```

Comment:
Some of this instructions look like should in Brev documentation, If any of these change in the future we will be out of sync, can we just link to their instructions?

Reply:
Ah, these existed in the original file that i edited.  Would you rather i remove them?  @jacobtomlinson for thoughts, as original author